src/World/createWeather.ts

```diff
@@ -74,16 +73,17 @@ setup.createWeather = (town, biome, weather) => {
         temperature: '',
         full: ''
       },
-      precipitation: '',
-      cloudIntensity: '',
-      precipitationLevel: seasonData.precipitationLevel,
-      precipitationIntensity: seasonData.precipitationIntensity
+      precipitation: 'no precipitation',
+      hasPrecipitation: false,
+      cloudIntensity: 'clear',
+      precipitationLevel: seasonData.precipitationLevel as PrecipitationLevels,
+      precipitationIntensity: seasonData.precipitationIntensity as PrecipitationIntensityLevels
```

Comment:
Do these need to be cast? They shouldn't, as long as the values being assigned *from* has the correct type.